scenarios/full_arc_create_validated_zero_arc_read/src/main.rs

```diff
@@ -0,0 +1,182 @@
+use anyhow::anyhow;
+use holochain_types::prelude::Record;
+use holochain_types::prelude::{ActionHash, Timestamp};
+use holochain_wind_tunnel_runner::prelude::*;
+use holochain_wind_tunnel_runner::scenario_happ_path;
+use std::collections::HashSet;
+use std::time::SystemTime;
+use timed_and_validated_integrity::TimedSampleEntry;
+
+const RECORD_OPEN_CONNECTIONS_PERIOD_MS: i64 = 3_000;
+
+#[derive(Debug, Default)]
+struct ScenarioValues {
+    sent_actions_count: u32,
+    seen_actions: HashSet<ActionHash>,
+    open_connections_last_recorded: Option<Timestamp>,
+}
+
+impl UserValuesConstraint for ScenarioValues {}
+
+fn agent_setup(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
+) -> HookResult {
+    if ctx.assigned_behaviour() == "zero" {
+        ctx.get_mut()
+            .holochain_config_mut()
+            .with_target_arc_factor(0);
+    }
+    start_conductor_and_configure_urls(ctx)?;
+    install_app(
+        ctx,
+        scenario_happ_path!("timed_and_validated"),
+        &"timed_and_validated".to_string(),
+    )?;
+
+    Ok(())
+}
+
+fn record_open_connections_if_necessary(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<ScenarioValues>>,
+    arc: String,
+) -> anyhow::Result<()> {
+    let now = Timestamp::now();
+    let should_record = match ctx.get_mut().scenario_values.open_connections_last_recorded {
```

Comment:
Using this new logic now to produce less metrics, as proposed in [this conversation](https://github.com/holochain/wind-tunnel/pull/345#discussion_r2556068244) of a previous PR.